widget/popup.go

```diff
@@ -14,8 +14,9 @@ import (
 type PopUp struct {
 	BaseWidget
 
-	Content fyne.CanvasObject
-	Canvas  fyne.Canvas
+	Content        fyne.CanvasObject
+	Canvas         fyne.Canvas
+	WithoutPadding bool
```

Comment:
In Canvas we use `Padded` - helps to differentiate from the theme `Padding`.
I wonder if this should be consistent with that, such as `NotPadded` or `UnPadded`.
Brain going slow but I think you get the gist?

Reply:
It's not entirely clear what will use this as I think it's just called from tests so far, I assume we can't flip the logic as that would remove padding from the &PopUp{} creations?

Reply:
> It's not entirely clear what will use this as I think it's just called from tests so far, I assume we can't flip the logic as that would remove padding from the &PopUp{} creations?

Yes, unfortunately this is true.
But it all will be consistent and good when we add elevation support some day.